image-builder-cli.spec

```diff
@@ -0,0 +1,97 @@
+# Do not build with tests by default
+# Pass --with tests to rpmbuild to override
+%bcond_with tests
+%bcond_with relax_requires
+
+# The minimum required osbuild version
+%global min_osbuild_version 129
+
+%global goipath         github.com/osbuild/image-builder-cli
+
+Version:        0
+
+%gometa
+
+%global common_description %{expand:
+A service for building customized OS artifacts, such as VM images and OSTree
+commits, that uses osbuild under the hood. Besides building images for local
+usage, it can also upload images directly to cloud.
+
+It is compatible with composer-cli and cockpit-composer clients.
+}
+
+Name:           image-builder-cli
+Release:        1%{?dist}
+Summary:        An image building service based on osbuild
+ExcludeArch:    i686 armv7hl
+
+# Upstream license specification: Apache-2.0
+License:        Apache-2.0
+URL:            %{gourl}
+Source0:        %{gosource}
+
+
+BuildRequires:  %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang}
+BuildRequires:  systemd
+BuildRequires:  krb5-devel
+BuildRequires:  python3-docutils
+BuildRequires:  make
+# Build requirements of 'theproglottis/gpgme' package
+BuildRequires:  gpgme-devel
+BuildRequires:  libassuan-devel
+# Build requirements of 'github.com/containers/storage' package
+BuildRequires:  device-mapper-devel
+%if 0%{?fedora}
+BuildRequires:  systemd-rpm-macros
```

Comment:
This is, in my opinion, not needed because we do not enable/disable/start/stop/restart any services as part of the RPM installation process.

Reply:
These are likely leftovers from the copy :)